main/exercise/ptest_agree_disagree.class.php

```diff
@@ -0,0 +1,293 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+use Chamilo\CourseBundle\Entity\CQuizAnswer;
+use ChamiloSession as Session;
+
+/**
+ * Class UniqueAnswer.
+ *
+ * This class allows to instantiate an object of type QUESTION_PT_TYPE_AGREE_OR_DISAGREE
+ * (Select the most agree and disagree options),
+ * extending the class question
+ *
+ * @author Jose Angel Ruiz (NOSOLORED)
+ *
+ * @package chamilo.exercise
+ */
+class PtestAgreeOrDisagree extends Question
+{
+    public $typePicture = 'ptaod.png';
+    public $explanationLangVar = 'PtestAgreeOrDisagree';
+
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->type = QUESTION_PT_TYPE_AGREE_OR_DISAGREE;
+        $this->isContent = $this->getIsContent();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createAnswersForm($form)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $nb_answers is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)